README.md

```diff
@@ -356,6 +356,7 @@ described in [RFC 8032]
 
 ## Markdown/Text Processors
  * [markd](https://github.com/icyleaf/markd) - Yet another markdown parser built for speed, Compliant to CommonMark specification
+ * [cr-cmark-gfm](https://github.com/amauryt/cr-cmark-gfm) -  Crystal C bindings for cmark-gfm to work with Commonmark and Github Flavored Markdown
```

Comment:
Technically doesn't this fit better under the `C Bindings` section?

Reply:
I suppose it could go there. But then wouldn't it be more difficult to discover? When I'm looking for markdown, I'm unlikely to go to the C Bindings section first.